acceptance/lib/pxp-agent/test_helper.rb

```diff
@@ -69,10 +69,11 @@ def show_pcp_logs_on_failure(&block)
 end
 
 # Some helpers for working with a pcp-broker 'lein tk' instance
-def run_pcp_broker(host)
-  on(host, "cd #{GIT_CLONE_FOLDER}/pcp-broker; export LEIN_ROOT=ok; lein tk </dev/null >/var/log/pcp-broker.log 2>&1 &")
-  assert(port_open_within?(host, PCP_BROKER_PORT, 60),
-         "pcp-broker port #{PCP_BROKER_PORT.to_s} not open within 1 minutes of starting the broker")
+def run_pcp_broker(host, instance=0)
+  host[:pcp_broker_instance] = instance
```

Comment:
right.  my thoughts were that the 'user' (test author) should never need to know what the ports are, just which instance they want to use.